### DISCUSSION TITLE
Prevent Comitup UI from appearing on touch screen

### DISCUSSION THREAD

**Discussion by jclarke0000** [outdoorbits/little-backup-box#327] [Q&A] [answered]
Hello.  I've created my own backup box with a touch screen, following the same install as in this YouTube video here:
https://www.youtube.com/watch?v=qAEebY8UlRc

I've noticed that when the device cannot connect to a known WiFi, I'm seeing the "Choose WiFi Connection" screen as presented by Comitup appear on the touch screen, and I cannot access the Little Backup Box UI until I do so.  If I cannot or don't want to connect to a WiFi, how do I bypass this screen and go directly to the Little Backup Box UI? Better yet, can this screen just be suppressed on the touch screen altogether?

Here's what I see when no known WiFi is present.  Even if I pick a WiFi to connect to, there's no way for me to enter a password, nor do I want to do so when I'm travelling.
![IMG_0116_1000px](https://github.com/user-attachments/assets/ace3ec8f-dbf1-4c7c-813d-723117621394)

**Comment by outdoorbits** [comment, marked as answer]
I'm traveling two more weeks, it's on my to-do list.

**Comment by jclarke0000** [comment]
Oh sweet!  Thank you

**Comment by outdoorbits** [comment]
solved by integrating comitup into the web UI
